### PR TITLE
Ports the blueshield

### PR DESCRIPTION
**Pull request by alexkar598**
:cl: alexkar598
add: Command now has a private body guard nammed the Blueshield
/:cl:

### CODE REVIEW

File: code/__DEFINES/jobs.dm [hunk 1]
```diff
@@ -12,6 +12,7 @@
 #define ROBOTICIST		(1<<8)
 #define AI_JF			(1<<9)
 #define CYBORG			(1<<10)
```
Comment (ghost) [inline code/__DEFINES/jobs.dm]: ~sunset_defines

File: code/game/gamemodes/changeling/changeling.dm [hunk 2]
```diff
@@ -11,7 +11,7 @@ GLOBAL_VAR(changeling_team_objective_type) //If this is not null, we hand our th
 	antag_flag = ROLE_CHANGELING
 	false_report_weight = 10
 	restricted_jobs = list("AI", "Cyborg")
-	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain")
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain","Blueshield")
```
Comment (ghost) [inline code/game/gamemodes/changeling/changeling.dm]: You can simply make an override for this in the sunset folder

File: code/game/gamemodes/clock_cult/clock_cult.dm [hunk 3]
```diff
@@ -134,7 +134,7 @@ Credit where due:
 	required_enemies = 4
 	recommended_enemies = 4
 	enemy_minimum_age = 14
-	protected_jobs = list("AI", "Cyborg", "Security Officer", "Warden", "Detective", "Head of Security", "Captain") //Silicons can eventually be converted
+	protected_jobs = list("AI", "Cyborg", "Security Officer", "Warden", "Detective", "Head of Security", "Captain","Blueshield") //Silicons can eventually be converted
```
Comment (ghost) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: You can simply make an override for this in the sunset folder

File: code/game/gamemodes/cult/cult.dm [hunk 4]
```diff
@@ -35,7 +35,7 @@
 	config_tag = "cult"
 	antag_flag = ROLE_CULTIST
 	false_report_weight = 10
-	restricted_jobs = list("Chaplain","AI", "Cyborg", "Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel")
+	restricted_jobs = list("Chaplain","AI", "Cyborg", "Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel","Blueshield")
```
Comment (ghost) [inline code/game/gamemodes/cult/cult.dm]: You can simply make an override for this in the sunset folder

File: code/game/gamemodes/devil/devil_game_mode.dm [hunk 5]
```diff
@@ -3,7 +3,7 @@
 	config_tag = "devil"
 	antag_flag = ROLE_DEVIL
 	false_report_weight = 1
-	protected_jobs = list("Lawyer", "Curator", "Chaplain", "Head of Security", "Captain", "AI")
+	protected_jobs = list("Lawyer", "Curator", "Chaplain", "Head of Security", "Captain", "AI","Blueshield")
```
Comment (ghost) [inline code/game/gamemodes/devil/devil_game_mode.dm]: You can simply make an override for this in the sunset folder

File: code/game/gamemodes/overthrow/overthrow.dm [hunk 6]
```diff
@@ -3,7 +3,7 @@
 	name = "overthrow"
 	config_tag = "overthrow"
 	antag_flag = ROLE_OVERTHROW
-	restricted_jobs = list("Security Officer", "Warden", "Detective", "AI", "Cyborg","Captain", "Head of Personnel", "Head of Security", "Chief Engineer", "Research Director", "Chief Medical Officer")
+	restricted_jobs = list("Security Officer", "Warden", "Detective", "AI", "Cyborg","Captain", "Head of Personnel", "Head of Security", "Chief Engineer", "Research Director", "Chief Medical Officer","Blueshield")
```
Comment (ghost) [inline code/game/gamemodes/overthrow/overthrow.dm]: You can simply make an override for this in the sunset folder

File: code/game/gamemodes/revolution/revolution.dm [hunk 7]
```diff
@@ -12,7 +12,7 @@
 	config_tag = "revolution"
 	antag_flag = ROLE_REV
 	false_report_weight = 10
-	restricted_jobs = list("Security Officer", "Warden", "Detective", "AI", "Cyborg","Captain", "Head of Personnel", "Head of Security", "Chief Engineer", "Research Director", "Chief Medical Officer")
+	restricted_jobs = list("Security Officer", "Warden", "Detective", "AI", "Cyborg","Captain", "Head of Personnel", "Head of Security", "Chief Engineer", "Research Director", "Chief Medical Officer","Blueshield")
 	required_players = 30
```
Comment (ghost) [inline code/game/gamemodes/revolution/revolution.dm]: You can simply make an override for this in the sunset folder

File: code/game/gamemodes/traitor/traitor.dm [hunk 8]
```diff
@@ -11,7 +11,7 @@
 	antag_flag = ROLE_TRAITOR
 	false_report_weight = 20 //Reports of traitors are pretty common.
 	restricted_jobs = list("Cyborg")//They are part of the AI if he is traitor so are they, they use to get double chances
-	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain")
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain","Blueshield")
```
Comment (ghost) [inline code/game/gamemodes/traitor/traitor.dm]: You can simply make an override for this in the sunset folder

File: code/modules/jobs/access.dm [hunk 9]
```diff
@@ -352,7 +352,7 @@
 	return list("Assistant", "Captain", "Head of Personnel", "Bartender", "Cook", "Botanist", "Quartermaster", "Cargo Technician",
 				"Shaft Miner", "Clown", "Mime", "Janitor", "Curator", "Lawyer", "Chaplain", "Chief Engineer", "Station Engineer",
 				"Atmospheric Technician", "Chief Medical Officer", "Medical Doctor", "Chemist", "Geneticist", "Virologist",
-				"Research Director", "Scientist", "Roboticist", "Head of Security", "Warden", "Detective", "Security Officer")
+				"Research Director", "Scientist", "Roboticist", "Head of Security", "Warden", "Detective", "Security Officer","Blueshield")
```
Comment (ghost) [inline code/modules/jobs/access.dm]: Forgot a ``// sunset -- add blueshit`

File: code/modules/jobs/jobs.dm [hunk 10]
```diff
@@ -44,7 +44,8 @@ GLOBAL_LIST_INIT(civilian_positions, list(
 	"Chaplain",
 	"Clown",
 	"Mime",
-	"Assistant"))
+	"Assistant",
+	"Blueshield"))
```
Comment (ghost) [inline code/modules/jobs/jobs.dm]: Forgot a comment

File: sunset/code/game/area/Space_Station_13_areas.dm [hunk 11]
```diff
@@ -1,3 +1,7 @@
 /area/sunset/genpop
 	name = "Genpop"
-	icon_state = "red"
\ No newline at end of file
+	icon_state = "red"
+
+/area/blueshield
```
Comment (ghost) [inline sunset/code/game/area/Space_Station_13_areas.dm]: Use `/area/sunset`

File: sunset/code/game/objects/items/devices/PDA/PDA_types.dm [hunk 12]
```diff
@@ -0,0 +1,4 @@
+/obj/item/device/pda/blueshield
```
Comment (ghost) [inline sunset/code/game/objects/items/devices/PDA/PDA_types.dm]: Change `/obj/item/device` to `/obj/item`.

File: sunset/code/game/objects/items/devices/radio/encryptionkey.dm [hunk 13]
```diff
@@ -0,0 +1,5 @@
+/obj/item/device/encryptionkey/heads/blueshield
```
Comment (ghost) [inline sunset/code/game/objects/items/devices/radio/encryptionkey.dm]: Change `/obj/item/device` to `/obj/item`.

File: sunset/code/game/objects/items/devices/radio/headset.dm [hunk 14]
```diff
@@ -0,0 +1,11 @@
+/obj/item/device/radio/headset/heads/blueshield
```
Comment (ghost) [inline sunset/code/game/objects/items/devices/radio/headset.dm]: Change `/obj/item/device` to `/obj/item`.

File: sunset/code/game/objects/items/devices/radio/headset.dm [hunk 15]
```diff
@@ -0,0 +1,11 @@
...
+	name = "\proper the blueshield's headset"
+	desc = "The headset of the Blueshield."
+	icon_state = "com_headset"
+	keyslot = new /obj/item/device/encryptionkey/heads/blueshield
+
+ /obj/item/device/radio/headset/heads/blueshield/alt
```
Comment (ghost) [inline sunset/code/game/objects/items/devices/radio/headset.dm]: Change `/obj/item/device` to `/obj/item`.

File: sunset/code/game/objects/structures/crates_lockers/closets/secure/blueshield.dm [hunk 16]
```diff
@@ -0,0 +1,17 @@
+/obj/structure/closet/secure_closet/blueshield
+	name = "\proper blueshield's locker"
+	req_access = list(ACCESS_BLUESHIELD)
+	icon_state = "blueshield"
```
Comment (ghost) [inline sunset/code/game/objects/structures/crates_lockers/closets/secure/blueshield.dm]: Spacing?

**Review by ghost — DISMISSED** [review]

File: code/controllers/subsystem/job.dm [hunk 17]
```diff
@@ -432,6 +432,10 @@ SUBSYSTEM_DEF(job)
 		to_chat(M, "<b>To speak on your departments radio, use the :h button. To see others, look closely at your headset.</b>")
 		if(job.req_admin_notify)
 			to_chat(M, "<b>You are playing a job that is important for Game Progression. If you have to disconnect, please notify the admins via adminhelp.</b>")
+		//sunset start
+		if(job.special_notice)
+		to_chat(M, "<span class='userdanger'>[job.special_notice]</span>")
```
Comment (ghost) [inline code/controllers/subsystem/job.dm]: Indent

File: sunset/code/game/objects/structures/crates_lockers/closets/secure/blueshield.dm (hunk 16 above)
Comment (alexkar598) [reply sunset/code/game/objects/structures/crates_lockers/closets/secure/blueshield.dm]: fixed

**Review by ghost — APPROVED** [review]
Looks good to me. Just port the sprites. DO NOT TOUCH /tg/ DMIs!!!!! DMIs are notorious for causing merge conflicts!

**Comment by alexkar598** [comment]
TODO: fix box sprite,fix locker sprite

**Review by ghost — APPROVED** [review]

**Pull request merged**